exonum-java-binding/core/src/test/java/com/exonum/binding/core/storage/database/MemoryDbIntegrationTest.java

```diff
@@ -52,6 +54,8 @@ void getSnapshotShallCreateNonNullSnapshot() throws Exception {
     }
   }
 
+  // FIXME: Disabled until ECR-3333
```

Comment:
1. Why, as we don't merge the fork but simply use it to access the state? 
2. The jira ref is unrelated.

Reply:
Hm, looks like it was disabled accidentally... Good observation, thanks!